src/robotide/editor/texteditor.py

```diff
@@ -973,6 +974,26 @@ def _enclose_text(open_symbol, value=''):
             close_symbol = open_symbol
         return open_symbol+value+close_symbol
 
+    def delete_row(self, event):
+        start, end = self._editor.GetSelection()
+        cursor = self._editor.GetCurrentPos()
+        ini_line = self._editor.LineFromPosition(start)
+        end_line = self._editor.LineFromPosition(end)
+        begpos = self._editor.PositionFromLine(ini_line)
+        endpos = self._editor.PositionFromLine(end_line+1)
+        self._editor.SelectNone()
+        self._editor.Remove(begpos, endpos)
```

Comment:
In my PR I changed this Remove, to LineDelete.

Reply:
What is the advantage of LineDelete over Remove?